src/frontend/sharedComponents/Button.tsx

```diff
@@ -59,9 +59,7 @@ const Button = ({
     <View style={touchableStyle as ViewStyle}>
       {
         typeof children === "string" ? (
-          <Text style={[styles.textBase, textStyle]}>
-            {children.toUpperCase()}
-          </Text>
+          <Text style={[styles.textBase, textStyle]}>{children}</Text>
```

Comment:
curious if this would "break" other parts of the app where we expect the copy to be all uppercase (although i agree with the change in general)

Reply:
Good Question. There might be some wierd casing with some button down the line. But I couldn't think of any thing that would actually break the app

Reply:
yeah i just mean "break" as in will change the expected UI somewhere. probably not a big deal but would be mindful of these kinds of changes

Reply:
OK, I will add this to the QA Script (specifically to take notice of any buttons that may be distorted or odd). We do have to change this eventually (for accessibility reasons). My mentality is to do it sooner rather then later so that we can catch bugs sooner.